sdk/nodejs/output.ts

```diff
@@ -70,7 +70,7 @@ class OutputImpl<T> implements OutputInstance<T> {
      */
     public readonly resources: () => Set<Resource>;
 
-    public readonly apply: <U>(func: (t: T) => Input<U>) => Output<U>;
+    public readonly apply: <U>(func: (t: T) => Input<U>, runWithUnknowns?: boolean) => Output<U>;
```

Comment:
I thought we decided not to expose this new overload publicly?  Can't we just leave this off here and the implementation just still handles it (and the one caller who uses it casts as needed)?

That said - is there any downside to exposing this publicly?  I assume it can be well-documented what the behaviour is in this case.  The only question I would have is whether we should accept an `ApplyOptions` here and have this be the single supported flag on that.  Not a huge deal though - we could expand to that pattern in the future if we need more flags.

Reply:
> I thought we decided not to expose this new overload publicly? Can't we just leave this off here and the implementation just still handles it (and the one caller who uses it casts as needed)?

Good catch. I'll fix this up.

> That said - is there any downside to exposing this publicly? I assume it can be well-documented what the behaviour is in this case. The only question I would have is whether we should accept an ApplyOptions here and have this be the single supported flag on that. Not a huge deal though - we could expand to that pattern in the future if we need more flags.

The only downsides are the requirement for additional documentation and the fact that first-class unknowns violate the stated typings. I do think that we should expose these in the future.